src/app/series/series.component.html

```diff
@@ -1,14 +1,12 @@
 <publish-hero blue=true>
-  <hero-title>
-    <h1 *ngIf="id">Edit Series</h1>
-    <h1 *ngIf="!id">Create Series</h1>
-  </hero-title>
-  <hero-info *ngIf="series">
+  <h1 class="hero-title" *ngIf="id">Edit Series</h1>
+  <h1 class="hero-title" *ngIf="!id">Create Series</h1>
+  <div class="hero-info" *ngIf="series">
     <h2>{{series.title || '(Untitled)'}}</h2>
     <p *ngIf="series?.isNew">Not saved</p>
     <p *ngIf="!series?.isNew">Last modified {{series.updatedAt | timeago}}</p>
-  </hero-info>
-  <hero-actions *ngIf="series">
+  </div>
+  <div class="hero-actions" *ngIf="series">
```

Comment:
They made [a change in RC6](https://github.com/angular/angular/issues/11251) that started throwing warnings for any unknown element names not covered by a component-selector.  So these `ng-content` selectors had to be switched to something else (at least for now - that github issue hints they may have other options down the line).